src/ConfigInterface.php

```diff
@@ -11,8 +11,9 @@
 namespace Laminas\Pimple\Config;
 
 use Pimple\Container;
+use Psr\Container\ContainerInterface;
 
 interface ConfigInterface
 {
-    public function configureContainer(Container $container) : void;
+    public function configureContainer(Container $container) : ContainerInterface;
```

Comment:
This **is** a BC break, as it's a change in signature. Implementations previously returning `void` are now invalid.

Granted, the primary _implementation_ is in this package; but technically speaking, this indicates a BC break nonetheless.